addons/binding/org.openhab.binding.kostal.inverter/src/main/java/org/openhab/binding/kostal/inverter/ChannelConfig.java

```diff
@@ -0,0 +1,13 @@
+package org.openhab.binding.kostal.inverter;
+
+public class ChannelConfig {
```

Comment:
Why do I need an author tag? I always think it is awkward in an open source project to have an author as over time many people might work on the file.